fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(data => this.setState({ users: data }))
+      .then(this.setState({ isLoading: false }))
+      .catch(err => console.error('error:', err));
```

Comment:
You can update multiple values in state with one `setState` call so this part should look like:

```JSX
fetch('https://jsonplaceholder.typicode.com/users')
      .then(res => res.json())
      .then(data => this.setState({ users: data, isLoading: false }))
      .catch(err => console.error('error:', err));
```